GetServiceByRevision/handler.ts

```diff
@@ -146,7 +154,7 @@ export function GetServiceByRevision(
   const handler = GetServiceByRevisionHandler(serviceModel);
   const middlewaresWrap = withRequestMiddlewares(
     RequiredParamMiddleware("serviceid", NonEmptyString),
-    RequiredParamMiddleware("version", NonNegativeInteger)
+    RequiredParamMiddleware("version", NonEmptyString)
```

Comment:
```suggestion
    RequiredParamMiddleware("version", IntegerFromString)
```

Reply:
Fixed in [7849fcb](https://github.com/pagopa/io-functions-assets/pull/2/commits/7849fcb0ecf2401a816bd7fe2f2f3265d5aa00a9)